apps/qubit/modules/default/actions/editAction.class.php

```diff
@@ -133,7 +133,7 @@ protected function processField($field)
 
         foreach ($this[$field->getName()] as $item)
         {
-          if (isset($value[$item->id]))
+          if (isset($value[$item->id]) && $value[$item->id])
```

Comment:
ah yeah, good call. I always forget how PHP has so many ways to check these sorts of things, all with their own nuances :|